HW_03_Task1/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace HW_03_Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello gues!");
+
+            addition();
+        }
+
+        static void addition()

```

Comment:
methods should start from upper case character

static void addition()  -> `static void Addition()`